sdk/translation/Azure.AI.Translation.Document/samples/CodeSamples/SampleTranslationWithAzureBlob.cs

```diff
@@ -0,0 +1,122 @@
+            var sourceContainerName = Environment.GetEnvironmentVariable("AZURE_STORAGE_SOURCE_CONTAINER_NAME");
+            var targetContainerName = Environment.GetEnvironmentVariable("AZURE_STORAGE_TARGET_CONTAINER_NAME");
+            var documentPath = Environment.GetEnvironmentVariable("AZURE_DOCUMENT_PATH");
+
+            // Create source and target storage containers
+            BlobServiceClient blobServiceClient = new BlobServiceClient(storageEndpoint, new StorageSharedKeyCredential(storageAccountName, storageAccountKey));
+            BlobContainerClient sourceContainerClient = await blobServiceClient.CreateBlobContainerAsync(sourceContainerName ?? "translation-source-container").ConfigureAwait(false);
+            BlobContainerClient targetContainerClient = await blobServiceClient.CreateBlobContainerAsync(targetContainerName ?? "translation-target-container").ConfigureAwait(false);
+
+            // Upload blob (file) to the source container
+            BlobClient srcBlobClient = sourceContainerClient.GetBlobClient(string.IsNullOrWhiteSpace(documentPath) ? Path.GetFileName(documentPath) : "example_source_document.txt");
+
+            if (!string.IsNullOrWhiteSpace(documentPath))
+            {
+                using (FileStream uploadFileStream = File.OpenRead(documentPath))
+                {
+                    await srcBlobClient.UploadAsync(uploadFileStream, true).ConfigureAwait(false);
+                }
+            }
+            else
+            {
+                await srcBlobClient.UploadAsync(new MemoryStream(Encoding.UTF8.GetBytes("Hello.\nThis is a testing text.")), true).ConfigureAwait(false);
+            }
+
+            Console.WriteLine($"Uploaded document {srcBlobClient.Uri} to source storage container");
+
+            // Generate SAS tokens for source & target
+            var srcSasUri = sourceContainerClient.GenerateSasUri(BlobContainerSasPermissions.List | BlobContainerSasPermissions.Read, DateTime.UtcNow.AddMinutes(30));
+            var tgtSasUri = targetContainerClient.GenerateSasUri(BlobContainerSasPermissions.Read | BlobContainerSasPermissions.Write | BlobContainerSasPermissions.Delete, DateTime.UtcNow.AddMinutes(30));
+
+            var documentTranslationClient = new DocumentTranslationClient(endpoint, new AzureKeyCredential(key));
+
+            var jobRequest = new List<DocumentTranslationInput>()
+            {
+                new DocumentTranslationInput(
+                    source: new TranslationSource(srcSasUri),
+                    targets: new List<TranslationTarget>()
+                        {
+                            new TranslationTarget(targetUri: tgtSasUri, languageCode: "es")
+                        }
+                    )
+            };
+
+            // Submit the translation job and wait for it to finish
+            var jobResult = await documentTranslationClient.StartTranslationAsync(jobRequest).ConfigureAwait(false);
```

Comment:
Python uses job :) .NET talks more about a translation and a translation operation.
For example: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/translation/Azure.AI.Translation.Document/tests/samples/Sample_StartTranslationAsync.cs#L34

Reply:
Changed to operation